app/timetables/models.py

```diff
@@ -1,3 +1,10 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from app.shared.models import CommonInfo
+
+class Course(CommonInfo, models.Model):
+    name = models.CharField(max_length=150)
```

Comment:
I don't think this class needs `time_created` and `time_modified`; so it does not need to inherit from `CommonInfo`.